### PR TITLE
Week 13 - project react native app - Nadia Mazzoni

### PR DESCRIPTION
**Pull request by Bluepoisson**
My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,26 +1,94 @@
-import React from 'react'
+import React, { useState } from 'react'
+
 import styled from 'styled-components/native'
+import { Entypo } from '@expo/vector-icons';
+
+
+import backgroundImage from './assets/banana.png';
+
+
```
Comment (EmOlsz) [inline App.js:9]: No need for so many line breaks ;)

File: App.js [hunk 2]
```diff
@@ -1,26 +1,94 @@
...
+`;
+
 
 const App = () => {
+  // const [enteredIntention, setEnteredIntention] = useState('');
+  const [dailyIntentions, setDailyIntentions] = useState([]);
+  const [isAddMode, setIsAddMode] = useState(false);
+
+  const addIntentionHandler = IntentionTitle => {       //getting user input and storing it
+    // console.log(enteredIntention);
```
Comment (EmOlsz) [inline App.js:48]: Let's remove all commented code

File: App.js [hunk 3]
```diff
@@ -1,26 +1,94 @@
...
+      setDailyIntentions(currentIntentions => [
+        ...currentIntentions,
+        { id: Math.random().toString(), value: IntentionTitle }
```
Comment (EmOlsz) [inline App.js:51]: Great job with immutable approach :)

File: components/IntentionInput.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Modal } from 'react-native';
+import styled from 'styled-components/native'
+
+import { Entypo } from '@expo/vector-icons';
+
+const ButtonCancel = styled.TouchableOpacity`
+  padding: 10px;
+  border-radius: 50px;
+  border: red;
```
Comment (EmOlsz) [inline components/IntentionInput.js:10]: I think you meant `border-color`

File: components/IntentionsItem.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: 24px;
+  color: #fff;
+  text-align: center;
+  margin-bottom: 5px;
+  padding: 10px;
+  border: #383E42;
+  border-radius: 10px;
+`;
+
+// const ButtonDelete = styled.TouchableOpacity`
```
Comment (EmOlsz) [inline components/IntentionsItem.js:16]: Commented code over here as well

File: components/IntentionsItem.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+//   padding: 10px;
+//   border-radius: 50px;
+//   border: red;
+//   margin-top: 15px;
+// `;
+
+const IntentionsItem = props => {
+    return (
+        <TouchableOpacity onPress={props.onDelete.bind(this, props.id)} >
+        {/* {dailyIntentions.map(intention => (
```
Comment (EmOlsz) [inline components/IntentionsItem.js:26]: And here

**Review by EmOlsz — CHANGES_REQUESTED** [review]

File: App.js (hunk 2 above)
Comment (Bluepoisson) [reply App.js:48]: Oh, yes! All comments removed now.

File: components/IntentionInput.js (hunk 4 above)
Comment (Bluepoisson) [reply components/IntentionInput.js:10]: Indeed I did. Didn't notice as no error message and the border was red.

File: components/IntentionsItem.js (hunk 5 above)
Comment (Bluepoisson) [reply components/IntentionsItem.js:16]: Gone!

File: App.js (hunk 3 above)
Comment (Bluepoisson) [reply App.js:51]: Thanks, it's that swooshing.

Comment (EmOlsz) [reply App.js:46]: ⭐️

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**